src/App.tsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.PureComponent<State> {
+  state: State = {
+    pressedKey: '',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handleKeySet);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleKeySet);
+  }
+
+  handleKeySet = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  handleKeyUnset = () => {
+    this.setState({ pressedKey: '' });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        {this.state.pressedKey ? (
+          <p
+            className="App__message"
+          >
+            The last pressed key is [
+            {pressedKey}
+            ]
+            <button
+              type="button"
+              className="delete"
+              aria-label="close"
+              onClick={() => this.handleKeyUnset()}
```

Comment:
I am not sure whether you had to add this button (it is not mentioned in the task), but it works, so let it be)